backend/src/index.js

```diff
@@ -1,48 +1,53 @@
-import express from 'express';
```

Comment:
Hey, I think you are running the wrong Linter.  
https://github.com/we-are-number-1/yumble/blob/main/CONTRIBUTING.md. We have decided to use ES Lint :).

Reply:
Hey, I noticed the same thing, do you think it would be possible to create pre-commit hooks to test the es linter?

Reply:
hmm, will consult with Alex, Good idea !

Reply:
ES Lint complains about express' Router function as it is using a capital letter. Should we just ignore these warnings or is there a way to suppress certain warnings?
<img width="706" alt="Screen Shot 2021-03-14 at 2 43 00 pm" src="https://user-images.githubusercontent.com/43579489/111054719-96543900-84d3-11eb-921e-49e81f1e11a9.png">
<img width="330" alt="Screen Shot 2021-03-14 at 2 43 12 pm" src="https://user-images.githubusercontent.com/43579489/111054722-9ce2b080-84d3-11eb-81fb-181ce2ea5f34.png">

Reply:
Leave it for now, I will ask Alex, he set up the linter. There seems to be a lot of other problems with it as well

Reply:
David has added an ESLint rule to ignore that specific issue